### PR TITLE
Adds ajax operator

### PR DESCRIPTION
**Pull request by paschalidi**
https://github.com/btroncone/learn-rxjs/issues/175

Let me know your thoughts and how I could improve this :) @btroncone

### CODE REVIEW

File: operators/creation/ajax.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# ajax
+
+#### signature: `ajax(subscribe: function)`
```
Comment (btroncone) [inline operators/creation/ajax.md]: I think this signature may be wrong, I believe it's `urlOrRequest: string | AjaxRequest`

Reference: https://github.com/ReactiveX/rxjs/blob/master/src/internal/observable/dom/AjaxObservable.ts#L155

**Comment by btroncone** [comment]
Your examples look great, thanks! I added a comment for one small tweak I think you need to make.

Also, just a gitbook convention, but can you also update the [`Summary.md`](https://github.com/btroncone/learn-rxjs/blob/master/SUMMARY.md) file to include the new `ajax` operator? This is what builds the table of contents.

Thanks again! 👍

**Comment by paschalidi** [comment]
@btroncone maybe have another look?

**Comment by btroncone** [comment]
Looks good! Just one more spot you will need to include the new operator, and I will get this merged. Check out the [README.md](https://github.com/btroncone/learn-rxjs/blob/master/operators/creation/README.md) file in the operators/creation section. Thanks again! 👍

**Comment by paschalidi** [comment]
@btroncone what holds us back from merging this one?

**Comment by btroncone** [comment]
Looks great, thanks! Merging now.

**Pull request merged**